.github/workflows/nightly-build-publish.yml

```diff
@@ -59,10 +59,10 @@ jobs:
           password: ${{ secrets.DOCKERHUB_PASSWORD }}
 
       - name: Build and push base images
-        run: /bin/bash arbitrary-users-patch/build_images.sh --push
+        run: /bin/bash -xe arbitrary-users-patch/build_images.sh --push
```

Comment:
Shouldn't we add flags in the script ?

Reply:
that's an option, yes. Wasn't sure if there was an intentional reason for having the script not fail for failures. If we want the -e insidethe script, I'm good with that.

Reply:
I don't see a case where we want to continue on failure inside the script

Reply:
arbitrary-users-patch scripts already set -e so I guess this is redundant.